addons/website_theme_install/models/ir_module_module.py

```diff
@@ -80,7 +81,7 @@ def write(self, vals):
                     if module.state == 'to upgrade' and request:
                         Website = self.env['website']
                         current_website = Website.get_current_website()
-                        websites_to_update = current_website in websites_to_update and current_website or Website
+                        websites_to_update = current_website if current_website in websites_to_update else Website
```

Comment:
NO DEBATE ABOUT IT !

Just for your information: https://www.pythoncentral.io/one-line-if-statement-in-python-ternary-conditional-operator/

Reply:
Thank for the link!

I just want this merged so I'm not going to debate it and I'll change it back.

That said in the link it specifies "and or" to be better to not evaluate twice the expression, which I agree.

But in this case, this it's just reading a variable so there is no performance impact and it is more readable with "if else"...